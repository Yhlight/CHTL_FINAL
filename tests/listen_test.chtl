html {
    head {
        title { text: "Listen Test"; }
    }
    body {
        button {
            id: "my-btn";
            text: "Click Me";
        }

        script {
            {{#my-btn}}->listen {
                click: () => {
                    alert("Button was clicked!");
                },
                mouseenter: () => {
                    console.log("Mouse entered button area.");
                }
            }
        }
    }
}
