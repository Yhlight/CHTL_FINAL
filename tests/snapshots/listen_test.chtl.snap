<html>
  <head>
    <title>
      Listen Test
    </title>
  </head>
  <body>
    <button id="my-btn">
      Click Me
    </button>
  </body>
</html>
